pytest_doctestplus/plugin.py

```diff
@@ -22,7 +22,7 @@
 
 _pytest_version = Version(pytest.__version__)
 PYTEST_GT_5 = _pytest_version > Version('5.9.9')
-PYTEST_GT_6_2 = _pytest_version > Version('6.2')
+PYTEST_GE_6_3 = _pytest_version >= Version('6.3')
```

Comment:
This was what I actually started with, but somehow didn't work.

Reply:
Yep, I see that :(